### DISCUSSION TITLE
Execution ends instead of listening to requests

### DISCUSSION THREAD

**Discussion by italomaia** [rwf2/Rocket#1981] [Questions]
Just tried the documentation example: https://pastebin.com/piCZvQpp
When I execute `cargo run` it simply calls index and ends the process. What could I be doing wrong here?

**Comment by somehowchris** [comment]
@italomaia well works for me, do you have a whole repository? on what are you running this? what rocket verion?....

**Reply by italomaia** [reply]
I recreated the rocket project and now it works. Do I know what caused the issue? No, but now it works. Can someone close this issue, please?

**Reply by somehowchris** [reply]
@italomaia that's how software development works 😆 

This isn't an issue, it's a dicussion feed on github.
